src/planner.c

```diff
@@ -33,8 +33,8 @@
 #include <parser/analyze.h>
 
 #include <catalog/pg_constraint.h>
-#include "compat.h"
 #include "compat-msvc-exit.h"
+#include "compat.h"
```

Comment:
Is there any particular reason for this reordering? I doubt `compat.h` depends on `compat-msvc-exit.h`. If not, then it breaks lexicographical order and is unnecessary change for the PR.

Reply:
Thank you for clarifying!